### ISSUE TITLE
Problema na listagem de roteiros

### ISSUE THREAD

**Issue by joaoxpedrox** [iptomar/PSI-Grupo-A-2019-2020#20]
Criei vários roteiros de teste ( utilizando a conta de Admin ) , e foi retornado sucesso. 
![image](https://user-images.githubusercontent.com/48485535/83522344-728dac80-a4d8-11ea-9fc1-8e7b79d9fb47.png)
Tudo ok até aqui. 

Ao lista-los no frontEnd não são visíveis. 
![image](https://user-images.githubusercontent.com/48485535/83522358-74f00680-a4d8-11ea-967e-71cbbea68821.png)

Nem ao listar no EndPoint (/routes/list) 
![image](https://user-images.githubusercontent.com/48485535/83522736-fe9fd400-a4d8-11ea-8741-40e684f9c96b.png)

Contudo, estão visíveis no campo do frontEnd. 
![image](https://user-images.githubusercontent.com/48485535/83522368-791c2400-a4d8-11ea-8bc9-55c664f89d5f.png)

**Comment by Difficulty01** [comment]
eles não são válidos

têm que ser validados depois de criados

mesmo criados por um utilizador

**Comment by Difficulty01** [comment]
mesmo que esse utilizador seja um administrador

**Comment by joaoxpedrox** [comment]
Sim. Mas por não estarem validados ainda, não deveriam aparecer no campo do FrontEnd para atribuição de pontos a roteiros.

**Comment by Difficulty01** [comment]
qual é o end point que estás a usar para a drop box?

**Comment by joaoxpedrox** [comment]
@aluno20451 @aluno20471  
Mas diria que estes 
![image](https://user-images.githubusercontent.com/48485535/83528033-863d1100-a4e0-11ea-974c-c0d1dfd23a0b.png)

**Comment by Difficulty01** [comment]
o primeiro retorna todos de um utilizador
validados ou não

o segundo
exactamente a mesma coisa

mas com os pontos

**Comment by joaoxpedrox** [comment]
Não fui eu que fiz isso @Difficulty01 
Só estou a alertar de que isso deve ser alterado. Não faz sentido teres isso ai a aparecer, quando não podes adicionar ai esses pontos.

**Comment by aluno20451** [comment]
Temos que fazer ainda um ecrã que mostre os roteiros só do utilizador, na parte de associar roteiros a meus pontos funciona corretamente
